src/oph/ehoks/oppija/schema.clj

```diff
@@ -14,14 +15,18 @@
 (s/defschema
   Jakolinkki
   "Tutkinnon osan jakolinkki"
-  {:share-id java.util.UUID
-   :tutkinnonosa-module-uuid java.util.UUID
-   :tutkinnonosa-tyyppi s/Str
-   :shared-module-uuid java.util.UUID
-   :shared-module-tyyppi s/Str
-   :voimassaolo-alku LocalDate
-   :voimassaolo-loppu LocalDate
-   :hoks-eid s/Str})
+  (s/->Either
```

Comment:
Jos tämä olisi skeema työpaikan toimijan näytölle niin tarvitaanko ollenkaan tätä Eitheriä? Jos jakolinkki on luotu niin sillä pitäisi löytyä dataa tai sitten se on vanhentunut jolloin palautetaan jokin ilmoitus mutta onko tarvetta missään tilanteessa palauttaa tyhjää?

Reply:
Tämä liittyi siihen kun haluttiin, että rajapinta ei palauta frontille not foundia vaan mieluummin tyhjän listan.

Reply:
Tuo tyhjän listan palautus taisi liittyä siihen jakolinkkien listaukseen mutta ei tähän työpaikan toimijan näyttöön. Eli jos ei ole tehty vielä yhtään riviä kantaan jaettavan osalta niin palautetaan vaan tyhjä lista